conf/igenomes.config

```diff
@@ -21,14 +21,19 @@ params {
             chr_dir               = "${params.igenomes_base}/Homo_sapiens/GATK/GRCh37/Sequence/Chromosomes"
             dbsnp                 = "${params.igenomes_base}/Homo_sapiens/GATK/GRCh37/Annotation/GATKBundle/dbsnp_138.b37.vcf"
             dbsnp_tbi             = "${params.igenomes_base}/Homo_sapiens/GATK/GRCh37/Annotation/GATKBundle/dbsnp_138.b37.vcf.idx"
+            dbsnp_vqsr            = '--resource:dbsnp,known=false,training=true,truth=false,prior=2.0 dbsnp_138.b37.vcf'
```

Comment:
Phil already uploaded the zipped files, I openend a separate PR for updating the config. So the values here just need updating and the new files you are adding :)